tests/e2e.sh

```diff
@@ -21,11 +21,11 @@ KUBERNETES_VERSION=v1.10.0
 KUBE_STATE_METRICS_LOG_DIR=./log
 KUBE_STATE_METRICS_IMAGE_NAME='quay.io/coreos/kube-state-metrics'
 PROMETHEUS_VERSION=2.0.0
-E2E_SETUP_MINIKUBE=${E2E_SETUP_MINIKUBE-yes}
-E2E_SETUP_KUBECTL=${E2E_SETUP_KUBECTL-yes}
-E2E_SETUP_PROMTOOL=${E2E_SETUP_PROMTOOL-yes}
-MINIKUBE_DRIVER=${MINIKUBE_DRIVER:-none}
-SUDO=${SUDO-sudo}
+E2E_SETUP_MINIKUBE=${E2E_SETUP_MINIKUBE:-}
+E2E_SETUP_KUBECTL=${E2E_SETUP_KUBECTL:-}
+E2E_SETUP_PROMTOOL=${E2E_SETUP_PROMTOOL:-}
+MINIKUBE_DRIVER=${MINIKUBE_DRIVER:-virtual}
```

Comment:
I think you meant this to be `virtualbox`, no?

Reply:
yes 🤦‍♀️